src/Gui/Inventor/Draggers/SoTransformDragger.cpp

```diff
@@ -887,3 +828,24 @@ bool SoTransformDragger::isHiddenRotationZ()
     SoSwitch* sw = SO_GET_ANY_PART(this, "zRotatorSwitch", SoSwitch);
     return (sw->whichChild.getValue() == SO_SWITCH_NONE);
 }
+
+void SoTransformDragger::setupTranslationDraggers()
+{
+    setupTranslationDragger("xTranslatorDragger", &xAxisLabel, translationIncrementCountX, SbVec3f(0.0,  0.0, -1.0));
+    setupTranslationDragger("yTranslatorDragger", &yAxisLabel, translationIncrementCountY, SbVec3f(0.0,  1.0,  0.0));
+    setupTranslationDragger("zTranslatorDragger", &zAxisLabel, translationIncrementCountZ, SbVec3f(1.0,  0.0,  0.0));
+}
+
+void SoTransformDragger::setupTranslationDragger(const std::string &name, SoSFString *label, SoSFInt32 &incrementCount, const SbVec3f rotAxis)
```

Comment:
`*` and `&` should be aligned to the left (type) as this is the style that is dominant in FreeCAD source

Reply:
fixed

Reply:
Actually, I thought that there will be some auto format thingy that will handle this lol.

Reply:
Should be, but for that file it probably is not enabled (yet).